### ISSUE TITLE
[Bug]: <Problem in ad_diff_conductance (shapes are not consistent) >

### ISSUE THREAD

**Issue by Ni2M** [PMEAL/OpenPNM#1568]
**Describe the bug**
line https://github.com/PMEAL/OpenPNM/blob/f49c0bb5eb04926084e80b2d3cd10b6d1e2723f0/openpnm/models/physics/ad_dif_conductance.py#L88

Apparently the shapes of the denominator and nominator are different. using np.tile will make the conductance array shape, but the rates shape is not consistent with g.

**To Reproduce**
Steps to reproduce the behavior, ideally a code snippet that only reproduces the bug:
```python
<import openpnm as op
import numpy as np
net = op.network.Cubic(shape=[10, 10, 1])
geom = op.geometry.StickAndBall(network=net)
air = op.phases.Air(network=net, name="air")
water = op.phases.Water(network=net, name="water")
x, y, z = net["pore.coords"].T
ps_water = net.Ps[x <= 30]
ps_air = np.setdiff1d(net.Ps, ps_water)
ts_water = net.find_neighbor_throats(pores=ps_water, mode="xnor")
ts_air = net.find_neighbor_throats(pores=ps_air, mode="xnor")
ts_interface = net.find_neighbor_throats(pores=ps_water, mode="xor")
# Define multiphase and set phase occupancy
mphase = op.phases.MultiPhase(network=net, phases=[air, water], name="mphase")
mphase._set_automatic_throat_occupancy()
mphase.set_occupancy(air, pores=ps_air, throats=ts_air)
mphase.set_occupancy(water, pores=ps_water, throats=ts_water)
phys = op.physics.Standard(network=net, phase=mphase, geometry=geom)
mdiff = op.models.physics.diffusive_conductance.multiphase_diffusion
phys.add_model(propname="throat.diffusive_conductance", model=mdiff)
phys.regenerate_models()
>
```

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by Ni2M** [comment]
I will take a further look into the problem and fixing it, but I opened the issue to make sure we don't forget it.

**Comment by ma-sadeghi** [comment]
@Ni2M Thanks for reporting it. This is fixed in PR #1569

**Comment by Ni2M** [comment]
@ma-sadeghi  Thank your very much for fixing and following up.
